borg/archive.py

```diff
@@ -420,6 +409,17 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
                 os.lchflags(path, item[b'bsdflags'])
             except OSError:
                 pass
+        xattrs = item.get(b'xattrs', {})
+        for k, v in xattrs.items():
+            try:
+                xattr.setxattr(fd or path, k, v, follow_symlinks=True)
```

Comment:
Duh? Why the symlinks semantics change all of a sudden? Looks to introduce a potential race condition.